packages/ozone/src/db/migrations/20240116T085607200Z-communication-template.ts

```diff
@@ -0,0 +1,23 @@
+import { Kysely } from 'kysely'
+
+export async function up(db: Kysely<unknown>): Promise<void> {
+  await db.schema
```

Comment:
Open question: should we seed any default templates in here?

Reply:
I think the idea was to not expose any of the templates we use internally. So, we could seed with a sample template just as a starting point (which seems a bit unnecessary) but we won't be seeding with our current templates for sure.